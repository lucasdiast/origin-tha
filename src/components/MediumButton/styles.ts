import styled from 'styled-components';

export const Container = styled.a`
  display: block;
  left: 0;
  bottom: 0;
  right: 0;
  font-size: 16px;
  max-width: 296px;
  padding: 10px;
  border-radius: 32px;
  line-height: 20px;
  margin: 0 auto;
  text-align: center;
  text-decoration: none;
  cursor: pointer;
  color: ${(props) => props.theme.colors.white};
  background-color: ${(props) => props.theme.colors.originBlue};
`;
