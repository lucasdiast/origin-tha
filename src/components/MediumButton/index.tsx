import React, { ReactElement, ReactChild } from 'react';
import { Container } from './styles';

interface AuxProps {
  children: ReactChild;
  link: string;
}

const MediumButton = ({ children, link }: AuxProps): ReactElement => (
  <Container className="big-button" href={link}>
    {children}
  </Container>
);

export default MediumButton;
