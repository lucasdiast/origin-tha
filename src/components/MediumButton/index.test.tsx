import { shallow } from 'enzyme';
import MediumButton from './index';

describe('MediumButton', () => {
  it('renders correctly', () => {
    const component = shallow(<MediumButton>Text</MediumButton>);

    expect(component).toMatchSnapshot();
  });
});
