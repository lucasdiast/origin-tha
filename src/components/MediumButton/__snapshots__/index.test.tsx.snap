// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BigButton renders correctly 1`] = `ShallowWrapper {}`;

exports[`MediumButton renders correctly 1`] = `ShallowWrapper {}`;
